sips/sip-02X/sip-02X-multisignature-transaction-signing-order-independence.md

```diff
@@ -0,0 +1,61 @@
+# Preamble
+
+SIP Number: 02X
+
+Title: Multisignature transaction signing order independence
+
+Author: Vladislav Bespalov <https://github.com/fess-v>
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Draft
+
+Created: 30 April 2023
+
+License: CC0-1.0
+
+Sign-off: -
+
```

Comment:
it would be good to add the relevant _Layer_ field affected by this proposal. Please refer to https://github.com/stacksgov/sips/blob/main/sips/sip-000/sip-000-stacks-improvement-proposal-process.md